README.md

```diff
@@ -62,6 +62,9 @@ Or on CodeSandBox at the following url: <a href="https://codesandbox.io/s/04wxlo
 | E key                      | Last slide                                          |
 | SpaceBar                   | When `autoplay={true}` pauses and unpauses carousel |
 
+
+- Keyboard shortcuts are disabled as a default. To enable them set the props to `true`.
```

Comment:
You might want to add which prop needs to be set to true. I don't think it's part of this section already.